### PR TITLE
white screen: normal text has black color

### PR DESCRIPTION
**Pull request by mojowebdev**
Feature request: #263 

## Colored output for white terminals

-W option to enable black color for normal text. 

white-screen

### CODE REVIEW

**Comment by czardoz** [comment]
thanks, @mojowebdev !

File: Gruntfile.js [hunk 1]
```diff
@@ -56,6 +61,7 @@ module.exports = function(grunt) {
 	grunt.registerTask('prepare_release', ['lint', 'mochaTest', 'jsdoc']);
 	grunt.registerTask('default', ['mochaTest', 'jsdoc', 'jshint']);
 	grunt.registerTask('test', ['mochaTest', 'jshint','run:*']);
-	grunt.registerTask('integ_test',['run:*'])
+	grunt.registerTask('integ_test',['run:*']);
 	grunt.registerTask('docs', 'jsdoc');
+	grunt.registerTask('test_white_screen',['run:whiteScreenTest']);
```
Comment (czardoz) [inline Gruntfile.js]: Instead of adding a new task, could you add this to the ``integ_test`` task?

**Comment by mojowebdev** [comment]
@czardoz Please check.

**Comment by czardoz** [comment]
Looks good, merging.

**Pull request merged**